test/spec/SpecRunnerUtils.js

```diff
@@ -995,15 +995,21 @@ define(function (require, exports, module) {
      * Simulate key event. Found this code here:
      * http://stackoverflow.com/questions/10455626/keydown-simulation-in-chrome-fires-normally-but-not-the-correct-key
      *
-     * TODO: need parameter(s) for modifier keys
-     *
      * @param {Number} key Key code
      * @param (String) event Key event to simulate
      * @param {HTMLElement} element Element to receive event
+     * @param {KeyboardEventInit} options Optional arguments for key event
      */
-    function simulateKeyEvent(key, event, element) {
-        var doc = element.ownerDocument,
-            oEvent = doc.createEvent('KeyboardEvent');
+    function simulateKeyEvent(key, event, element, options) {
+        var doc = element.ownerDocument;
+        var defaultOptions = {
+        		view: doc.defaultView,
+        		bubbles: true,
+        		cancelable: true,
+        		keyIdentifier: key
+        };
+        var newOptions = Object.assign({}, options, defaultOptions);
```

Comment:
It was brought up in another PR that using `Object.assign` actually brokes the Linux version due to using an age old CEF which doesn't support it 😢 

If you could change this to the previous format then this one is ready to be merged